wiki/Welcome/id.md

```diff
@@ -1,49 +1,45 @@
+- [Pendaftaran](/wiki/Registration)
 - [Peraturan](/wiki/Rules)
-- [Pertanyaan yang Sering Diajukan](/wiki/FAQ)
-- [Mode Permainan](/wiki/Game_mode)
+- [Pertanyaan yang sering diajukan](/wiki/FAQ)
+- [Jenis permainan](/wiki/Game_mode)
   - ![Ikon osu!standard](/wiki/shared/mode/osu.png) [osu!standard](/wiki/osu!standard)
   - ![Ikon osu!taiko](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/osu!taiko)
   - ![Ikon osu!catch](/wiki/shared/mode/catch.png) [osu!catch](/wiki/osu!catch)
   - ![Ikon osu!mania](/wiki/shared/mode/mania.png) [osu!mania](/wiki/osu!mania)
 - [Multi](/wiki/Multi)
-- [Kriteria Ranking](/wiki/Ranking_Criteria)
+- [Kriteria ranking](/wiki/Ranking_Criteria)
   - ![Ikon osu!standard](/wiki/shared/mode/osu.png) [osu!standard](/wiki/Ranking_Criteria/osu!standard)
   - ![Ikon osu!taiko](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Ranking_Criteria/osu!taiko)
   - ![Ikon osu!catch](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Ranking_Criteria/osu!catch)
   - ![Ikon osu!mania](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Ranking_Criteria/osu!mania)
 
 ## Beatmapping
 
-*Lihat juga: [Beatmapping](/wiki/Beatmapping).*
+*Laman utama: [Beatmapping](/wiki/Beatmapping).*
 
-Beatmapping adalah proses pemain yang akan menciptakan beatmap. Proses ini meliputi: pemilihan musik, mengatur waktu dan memetakan beatmap, dan menguji beatmap. Mungkin juga termasuk: menambahkan video, menambahkan [storyboard](/wiki/Storyboarding), dan/atau menambahkan [skin kustom](/wiki/Skinning). Semua itu dapat dilakukan oleh pembuat beatmap itu sendiri atau dengan pemain lain, kadang-kadang disebut juga *storyboarders*, *skinners*, dan/atau *guest creators*, tergantung pada apa yang mereka lakukan.
+Beatmapping adalah proses pembuatan beatmap yang dilakukan oleh mapper. Proses ini meliputi: pemilihan musik, mengatur tempo (timing) dan pemetaan beatmap, dan pengujian beatmap. Topik yang juga berhubungan: menambahkan video, menambahkan [storyboard](/wiki/Storyboarding), dan/atau menambahkan [skin kustom](/wiki/Skinning). Semua itu dapat dilakukan oleh mapper itu sendiri atau berkolaborasi dengan pemain lain, pemain yang memiliki kemampuan berikut dikenal juga dengan *storyboarders*, *skinners*, dan/atau *guest creators*, tergantung pada apa yang mereka lakukan.
 
 ## Modding
 
-*Lihat juga: [Modding](/wiki/Modding).*
+*Laman utama: [Modding](/wiki/Modding).*
 
-Modding adalah proses pengguna meninjau (atau biasa disebut "modders") beatmap pembuat dalam tahap pending (atau work in progress/help). Modding memainkan peran besar dalam pengontrolan kualitas, karena hal ini memungkinkan pembuat beatmap untuk memperbaiki masalah di beatmap yang mereka buat.
+Modding adalah proses pengguna dalam meninjau (atau biasa disebut "modders") beatmap seorang mapper yang beatmap-nya masih berstatus pending (atau work in progress/help). Modding memainkan peran besar dalam pengontrolan kualitas, dengan ini mapper dapat menyelesaikan yang masih bersarang di beatmap mereka.
```

Comment:
Modding adalah proses pengguna dalam meninjau (atau biasa disebut "modders") beatmap seorang mapper yang beatmap-nya masih berstatus pending (atau work in progress/help). Modding memainkan peran besar dalam meningkatkan kualitas, dengan ini mapper dapat menyelesaikan masalah yang ada di beatmap mereka.